policyengine_us/parameters/gov/states/nd/tax/income/rates/head_of_household.yaml

```diff
@@ -4,37 +4,48 @@ brackets:
       2021-01-01: 0
     rate:
       2021-01-01: 0.011
+      2023-01-01: 0
   - threshold:
       2021-01-01: 54_300
       2022-01-01: 55_900
+      2023-01-01: 59_950
     rate:
       2021-01-01: 0.0204
+      2023-01-01: 0.0195
   - threshold:
       2021-01-01: 140_200
       2022-01-01: 144_400
+      2023-01-01: 250_550
     rate:
       2021-01-01: 0.0227
+      2023-01-01: 0.0250
   - threshold:
       2021-01-01: 226_950
       2022-01-01: 233_750
+      2023-01-01: .inf
     rate:
       2021-01-01: 0.0264
+      2023-01-01: 0.0250
   - threshold:
       2021-01-01: 445_000
       2022-01-01: 458_350
+      2023-01-01: .inf
     rate:
       2021-01-01: 0.029
+      2023-01-01: 0.0250
 
 metadata:
   threshold_unit: USD
   rate_unit: /1
   threshold_period: year
   type: marginal_rate
-  label: ND income tax rates for head-of-household filing units
+  label: North Dakota income tax rates for head-of-household filing units
   reference:
   - title: 2021 ND income tax instructions
-    href: https://www.tax.nd.gov/sites/www/files/documents/forms/2021-individual-income-tax-booklet.pdf#page=34
+    href: https://www.tax.nd.gov/sites/www/files/documents/forms/individual/2021-iit/individual-income-tax-booklet-2021.pdf#page=34
   - title: 2022 ND income tax instructions
-    href: https://www.tax.nd.gov/sites/www/files/documents/forms/2022-individual-income-tax-booklet.pdf#page=34
+    href: https://www.tax.nd.gov/sites/www/files/documents/forms/individual/2022-iit/2022-individual-income-tax-booklet.pdf#page=34
+  - title: 2023 ND income tax instructions
+    href: https://www.tax.nd.gov/sites/www/files/documents/forms/individual/2023-iit/2023-individual-income-tax-booklet.pdf#page=30
   - title: North Dakota legal code 57-38-30.3. Individual, estate, and trust income tax.
     href: https://ndlegis.gov/cencode/t57c38.pdf#nameddest=57-38-30p3#page=29
```

Comment:
this is the whole legal code pdf, please use this in all files - **with the right page numbers and subsections**